.github/workflows/build.yml

```diff
@@ -0,0 +1,50 @@
+name: Build CI
```

Comment:
This and the other file are currently in the workflows directory for the _cookiecutter repo_, not the cookiecutter generated project. 

@dherrada please move these changes into this directory so that they appear as change to the workflow files that Scott added:

https://github.com/adafruit/cookiecutter-adafruit-circuitpython/tree/master/%7B%7B%20cookiecutter.library_name%20%7D%7D/.github/workflows